src/utils.ts

```diff
@@ -317,9 +348,9 @@ export async function getScores(
     );
   }
 
-  const url = new URL(scoreApiUrl);
-  url.pathname = '/api/scores';
-  scoreApiUrl = url.toString();
+  const urlObject = new URL(scoreApiUrl);
+  urlObject.pathname = '/api/scores';
+  const { url, headers } = formatScoreAPIUrl(urlObject.toString());
```

Comment:
We could move this pathname assignation, and delegate it to `formatScoreAPIUrl` as an optional second args

```suggestion
  const { url, headers } = formatScoreAPIUrl(scoreApiUrl, '/api/scores');
```

And add

```
if (pathname) urlObject.pathname = pathname;
```

inside `formatScoreAPIUrl`